PWGCF/Flow/Tasks/resonancesGfwFlow.cxx

```diff
@@ -763,6 +1012,41 @@ struct ResonancesGfwFlow {
     fillResoProfile(corrconfigs.at(22), HIST("Lambdav24pt"), cent, fLambdaMassAxis);
     fillResoProfile(corrconfigs.at(23), HIST("AnLambdav24pt"), cent, fLambdaMassAxis);
 
+    // bootstraping
+    if (cfgUseBootStrap) {
+      TRandom3* rand = new TRandom3(0);
```

Comment:
This will leak a `TRandom3` object per collision 
Why not using `fRndm` which is correctly allocated at the task level?